src/notification.c

```diff
@@ -234,129 +234,184 @@ check_value_type (const char *key,
 }
 
 static gboolean
-check_priority (GVariant *value,
-                GError **error)
+parse_priority (GVariantBuilder  *builder,
+                GVariant         *value,
+                GError          **error)
 {
   const char *priorities[] = { "low", "normal", "high", "urgent", NULL };
+  const char *priority;
 
   if (!check_value_type ("priority", value, G_VARIANT_TYPE_STRING, error))
     return FALSE;
 
-  if (!g_strv_contains (priorities, g_variant_get_string (value, NULL)))
-    {
-      g_set_error (error,
-                   XDG_DESKTOP_PORTAL_ERROR,
-                   XDG_DESKTOP_PORTAL_ERROR_INVALID_ARGUMENT,
-                   "%s not a priority", g_variant_get_string (value, NULL));
-      return FALSE;
-    }
+  priority = g_variant_get_string (value, NULL);
+
+  if (g_strv_contains (priorities, priority))
+    g_variant_builder_add (builder, "{sv}", "priority", value);
+  else
+    g_debug ("Unsupported priority %s filtered from notification", priority);
```

Comment:
same question here: ignoring unsupported properties is fine and all, but when the data is bogus, it should result in an error, no?

Reply:
well if we want to later add more possible priorities...

Reply:
That's why we the portals are versioned.